composer.json

```diff
@@ -36,7 +36,7 @@
     "symfony/validator": "^5.1",
     "symfony/yaml": "^5.1@dev",
     "typhonius/acquia-logstream": "dev-master",
-    "typhonius/acquia-php-sdk-v2": "dev-master as 2.0.13",
+    "typhonius/acquia-php-sdk-v2": "^2.0.14",
```

Comment:
Changing to a stable release was necessary to fix a Travis CI error (not clear on the precise cause). And anyway, stable versions are better, and Adam is pretty responsive about cutting stable releases.